src/agent/communicator/src/http_client.cpp

```diff
@@ -125,7 +124,7 @@ namespace http_client
 
```

Comment:
In line https://github.com/wazuh/wazuh-agent/pull/355/files#diff-301a5da8240724f147370f285f68ff252a29505ea09942939dbec96f24034596R120, make `req` `const`.